src/main/java/ch/njol/skript/expressions/ExprRemainingAir.java

```diff
@@ -52,8 +58,40 @@ protected String getPropertyName() {
 	}
 	
 	@Override
-	public Timespan convert(final LivingEntity e) {
-		return Timespan.fromTicks(e.getRemainingAir());
+	public Timespan convert(final LivingEntity entity) {
+		return Timespan.fromTicks_i(entity.getRemainingAir());
+	}
+	
+	@Nullable
+	@Override
+	public Class<?>[] acceptChange(Changer.ChangeMode mode) {
+		return CollectionUtils.array(Timespan.class);
+	}
+	
+	@SuppressWarnings("null")
+	@Override
+	public void change(Event event, @Nullable Object[] delta, Changer.ChangeMode mode) {
+		long ticks = ((Timespan)delta[0]).getTicks_i();
+		switch (mode) {
+			case ADD:
+				for (LivingEntity entity : getExpr().getArray(event))
+					entity.setRemainingAir(entity.getRemainingAir() + (int) ticks);
+				break;
+			case REMOVE:
+				for (LivingEntity entity : getExpr().getArray(event))
+					entity.setRemainingAir(entity.getRemainingAir() - (int) ticks);
+				break;
+			case SET:
+				for (LivingEntity entity : getExpr().getArray(event))
+					entity.setRemainingAir((int) ticks);
+				break;
+			case DELETE:
+			case REMOVE_ALL:

```

Comment:
remove all != delete.

Reply:
In some cases, it is. Removing all air from lungs would be pretty much same than deleting it, right?

Reply:
I feel it would

Reply:
The thing is that it would make a pretty weird syntax, as you can see:
```
remove all 5 from remaining air of player
```

Reply:
`remove all the remaining air from the player`

The number isn't present, that would be `remove 5 from remaining air of player`

Reply:
the number has to be present as it isn't optional. I think you're just misunderstanding the functionality of the REMOVE_ALL changer here.